src/vga/src/lib.rs

```diff
@@ -130,6 +137,17 @@ impl VgaBuffer {
 
         self.flush();
     }
+
+    fn set_cursor(&self, position: u16) {
+        unsafe {
+            // Set cursor low
+            outb(0x3D4, 0x0F);
+            outb(0x3D5, position as u8);
+            // Set cursor high
+            outb(0x3D4, 0x0E);
+            outb(0x3D5, (position >> 8) as u8);
+        }
+    }
 }
```

Comment:
Want to change this too... Use `self.position` instead of an arg and call it `update_cursor()` or maybe move it out of Vga impl